coarse/services/typo_grammar_agent.py

```
@@ -0,0 +1,15 @@
+from langchain.output_parsers import OutputFixingParser
+from langchain_core.output_parsers import JsonOutputParser
+from fastapi import UploadFile
```

Ruff (0.11.9)

`fastapi.UploadFile` imported but unused

Remove unused import: `fastapi.UploadFile`

(F401)